x-pack/plugins/siem/cypress/support/index.js

```diff
@@ -32,5 +32,10 @@ Cypress.on('uncaught:exception', err => {
   }
 });
 
+Cypress.on('window:before:load', win => {
```

Comment:
When does this hook fire? Is there a possibility of this interfering with other tests?

Reply:
Nope, this is not going to interfere with any test :)